lib/utils.pm

```diff
@@ -477,16 +482,28 @@ sub reboot_gnome {
     workaround_type_encrypted_passphrase;
 }
 
+=head2 poweroff
+
+    poweroff;
+
+Shutdown system from root console.
+=cut
+sub poweroff {
+    reboot('poweroff');
+}
+
 =head2 reboot
 
     reboot;
 
 Reboot system from root console.
 =cut
 sub reboot {
+    my ($action) = @_;
+    $action //= 'reboot';
```

Comment:
if you want it to be private then make it so. Create 'basecaasp' and put it there